removal_from_ansible.rst

```diff
@@ -0,0 +1,160 @@
+*****************************************************
```

Comment:
This needs an RST anchor. See https://docs.ansible.com/ansible/latest/dev_guide/style_guide/#adding-anchors. 
```suggestion
.. _removal_from_ansible::

*****************************************************
```